features/compact-editor.js

```diff
@@ -11,17 +11,18 @@
 
 var style = document.createElement("style");
 style.innerHTML = `
-.gui_target-wrapper_36Gbz {
-  padding-top: 0.25rem !important;
-}
```

Comment:
Why did you remove this?

Reply:
It looks bad and doesn't necessarily make the editor more compact

Reply:
Notice the space between the stage and the properties


Before

![image](https://user-images.githubusercontent.com/63302372/233858615-f7181a81-7bd8-473e-8383-6245bf0866e6.png)

After

![image](https://user-images.githubusercontent.com/63302372/233858686-39106344-2dfb-4989-ac33-dbbef5bbebe2.png)